src/routes/expensesRouter.js

```diff
@@ -0,0 +1,127 @@
+/* eslint-disable function-paren-newline */
+const { Router } = require('express');
+const { mockUsers } = require('./usersRouter.js');
+const expensesRouter = Router();
+const mockExpenses = new Map();
+
+const filterFunctions = {
```

Comment:
This filter could be inside your _Service_ file (the file you called _Controller_), would be a better place to store it